algorithms/loopclosure/loop-closure-handler/src/loop-detector-node.cc

```diff
@@ -648,10 +648,9 @@ bool LoopDetectorNode::findVertexInDatabase(
   *num_of_lc_matches = loop_closure::getNumberOfMatches(frame_matches_list);
 
   timing::Timer timer_compute_relative("lc compute absolute transform");
-  pose_graph::VertexId vertex_id_closest_to_structure_matches;
   bool ransac_ok = computeAbsoluteTransformFromFrameMatches(
       frame_matches_list, merge_landmarks, add_lc_edges, map, T_G_I,
-      inlier_constraint, &vertex_id_closest_to_structure_matches);
+      inlier_constraint, nullptr /**/);
```

Comment:
Either put in the comments the param name that is nullptr or don't use the empty comments?